book/typst/modules/module1-practice-problems.typ

```diff
@@ -1,56 +1,170 @@
     ],
     solution: [
     ],
     citation: [X],
   ),
   (
     statement: [
-      (Second question about modelling; similar to first but not a population model)
+      /*(Situation with a new model)
+      + Build a discrete model modelling population by year and compute pop after first three years
+      + By month
+      + Modify it to be a continuous model*/
+    ],
+  ),
+  (
+    statement: [
+      /*(Thrid question about modelling; similar to first but not a population model)*/
     ],
     solution: [
     ],
   ),
   (
     statement: [
-      (Third question about modelling; requires some real-world judgement, e.g. maybe something involving seasons)
+      (Forth question about modelling; requires some real-world judgement, e.g. maybe something involving seasons)
     ],
   ),
   (
     statement: [
-      (Give two diffeq models and ask which one grows faster initially; justify)
+      /*Give two diffeq models and ask which one
+      grows faster initially; justif*/
+      A biologist is measuring the population of bacteria in a petri dish. They are measuring two bacteria that grow at different rates. Population P_1 grows at a rate according to the equation $P_1'(t) = 3140 t P_1(t)$, and population P_2 grows at a rate according to the equation $P_2'(t) = 2880 t^2 P_2(t)$, where t is measured in days. The biologist biologist places exactly one cell simotaniously on each side of the petri dish at time $t=0$. The biologist know hypothesises that if one of the populations grows faster than the other of the entriety of the first day in the petry dish, then that population will eventually take over the petri dish. Will the biologist be able to test their hypothesises? If so, why? If not, why not?
     ],
   ),
   (
     statement: [
-      (Give a diffeq model with solutions that have maxes and mins. Ask what time(s) the solution could have a max/min (they need to use calculus to
-      determine where $y'(t)=0$ could occur))
+      /*(Give a diffeq model with solutions that have maxes and mins. Ask what time(s) the solution could have a max/min (they need to use calculus to
+      determine where $y'(t)=0$ could occur))*/
+
     ],
   ),
   (
     statement: [
-      (Show scratch-work of a "student" solving a diffeq by guess and check. Ask to write down the general solution (including indicating any parameters!))
+      /*(Show scratch-work of a "student" solving a diffeq by guess and check. Ask to write down the general solution (including indicating any parameters!))*/
+      A student is trying to solve a question about a car with the instruction that "The car is moving east at a rate propotional to the square of it's distance from the origin." 
```

Comment:
How about `A video game programmer is simulating a car based on the following instructions from the animator: ...`